Automattic/portkey-android
# Android 13 migration update compile and target sdk versions to 33

Updates compileSdkVersion and targetSdkVersion from 31 to 33 (Android 13)

Internal project thread: pbArwn-5IN-p2

Fixes: [#18096](https://github.com/wordpress-mobile/WordPress-Android/issues/18096)

To test

- Checkout this branch
- Use this branch through WP (uncomment localStoriesAndroidPath in local-builds.gradle)
- Try out adding a Story post (FAB > Story post) with an image and/or video
- Ensure, no issues or crashes
